### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by HunterX-7**
## Dear Code reviewer In this Project, my coding partner @cilfonegabriel and I:

- Add the following features to the project:

### Vet Clinic project requirements

- Take a look at the following database diagram for a clinic:
     
clinic_diagram

- Create a file named `schema_based_on_diagram.sql` where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Remember to add the FK indexes.

clinic_sql

## I Hope you have a great day and your support is much appreciated! 😄

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+create database clinic;
+
+create table if not exists patients(id int generated always as identity, name varchar(20), date_of_birth date, primary key(id));
+
+create table if not exists medical_histories(id int generated always as identity, admitted_at timestamp, patient_id int references patients(id), status varchar(20) primary key(id));
+
+create table if not exists medical_history_treatment(treatment_id references treatments(id), medical_histories_id references medical_histories(id));
```
Comment (Davidosky007) [inline schema_based_on_diagram.sql]: * Kindly add a Primary Key to this table as its best practice and It uniquely identifies each row of a table.

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @HunterX-7 @cilfonegabriel 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

* Well written PR description 👍 
* Good looking readme file 👍 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (HunterX-7) [reply schema_based_on_diagram.sql]: Thanks for your suggestions will kindly do that 😄

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @HunterX-7  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Made required changes 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**